whereby/browser-sdk
# Add rtcAnalytics slice and remaining rtcstats events

https://linear.app/whereby/issue/PAN-646/forward-remaining-events-from-rtcanalyticscustomeventsconfig

## Test plan
* run yarn dev
* Join room connection with local media story and join room
* Test that initial custom events all fire at start of session in ws messages
* Toggle mic, camera and screenshare actions
* Verify that custom events appear after toggling

<img width="1371" alt="Screenshot 2024-01-08 at 09 04 49" src="https://github.com/whereby/browser-sdk/assets/117101145/bd8008d6-6ef6-4ea6-8b74-7ccd64316f66">
